README.md

```diff
@@ -662,6 +662,11 @@ We aim for *complete* and *current* resources here whenever possible.
 * [Joey Hess' blog][129] - A collection of posts, many of which are about
   Haskell use. [CC-BY-4.0][136].
 
+#### Chrestomathy ####
```

Comment:
Is there an easier to understand word that could be used? Not everyone who stumbles upon this list is a native English speaker. I had to look up that word online :eyes: